ui/core/src/users/RelayQueryConfig.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * file: users/RelayQueryConfig.js
+ *
+ * Exports a RelayQueryConfig to be used with users/index.js
+ *
+ */
+import Relay from 'react-relay';
+
+const NodeQueries = {
```

Comment:
This should not be here anymore @rsjavier14, since you've moved them to **shared** dir